JavaWeaver/api/kadabra/analysis/energy/detectors/ExcessiveMethodCallsDetector.js

```diff
@@ -0,0 +1,422 @@
+laraImport("kadabra.analysis.energy.detectors.BaseDetector");
+
+const containsJP = (arr, jp) => arr.some((i) => i.same(jp));
+
+const tryGetMethodInfo = (arr, jp) => {
+  for (let j of arr) {
+    if (j.method.same(jp)) return j;
+  }
+  return null;
+};
+
+const addIfNew = (arr, jp) => {
+  if (!containsJP(arr, jp)) arr.push(jp);
+};
```

Comment:
If you use JS modules, there constants will be global only to this file, however, they will be globally accessible if a .lara file or a .js file that is not a module imports this file using laraImport().

Can these constants be moved as static members of the class ExcessiveMethodCallsDetector?